probe/ebpf/flow.c

```diff
@@ -307,20 +316,20 @@ static inline void update_metrics(struct __sk_buff *skb, struct flow *flow, __u6
 	}
 }
 
-static void fill_flow(struct __sk_buff *skb, struct flow *flow)
+static void fill_flow(struct __sk_buff *skb, struct flow *flow, __u64 tm)
 {
-	flow->ifindex = skb->ifindex;
+    flow->ifindex = skb->ifindex;
```

Comment:
The indentation still need to be fixed for both go and c

Reply:
done